core/src/main/java/org/kiwix/kiwixmobile/core/utils/dialog/AlertDialogShower.kt

```diff
@@ -51,6 +66,33 @@ class AlertDialogShower @Inject constructor(private val activity: Activity) :
               ?.invoke()
           }
         }
+        if (url != null) {
+          val textView = TextView(activity.baseContext)
+          textView.setPadding(5, 5, 5, 5)
+          textView.gravity = Gravity.CENTER
+          textView.setLinkTextColor(Color.BLUE)
+          textView.setOnLongClickListener {
+            val clipboard =
+              ContextCompat.getSystemService(activity.baseContext, ClipboardManager::class.java)
+            val clip = ClipData.newPlainText("External Url", "$url")
+            clipboard?.setPrimaryClip(clip)
+            Toast.makeText(
+              activity.baseContext,
+              "External Link Copied to Clipboard",
```

Comment:
hardcoded strings should be extracted I think

Reply:
Oh, yes will fix it!

Reply:
@4shutosh ,
done